Dockerfile

```diff
@@ -56,5 +56,6 @@ COPY --from=build /usr/src/app/node_modules ./node_modules
 COPY --from=build /usr/src/app/dist ./dist
 COPY --from=build /usr/src/app/package.json ./
 COPY --from=build /usr/src/app/prisma ./prisma
+COPY --from=build /usr/src/app/src/mailing/templates ./src/mailing/templates
```

Comment:
Okay if this works but would be better to have the path in env var so you are not mixing folder structure in Docker